plugins/outputs/newrelic/newrelic.go

```diff
@@ -115,9 +123,7 @@ func (nr *NewRelic) Write(metrics []telegraf.Metric) error {
 					mvalue = float64(1)
 				}
 			case string:
-				// Do not log everytime we encounter string
-				// we just skip
-				continue
+				tags[mname] = n
```

Comment:
This change is strange. The pattern for string now does not match the one of the other types. Are you sure this is correct or did you want to do `mname = n` here?

Reply:
This is correct, comment added to code to explain.